securedrop/source_templates/index.html

```diff
@@ -14,7 +14,7 @@
     {% endassets %}
   </head>
   <body>
-    <div class="js-warning warning">{{ gettext('<strong>We recommend turning the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="js-warning warning">{{ gettext('<strong>We recommend moving the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```

Comment:
s/Learn how to set it to safest/Learn how to set it to "Safest"
Quotes could be used or not, there is another "Safest" here.

Reply:
done